script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -74,9 +74,9 @@ Thus, the class has become a mysterious space divided into simple factions of 'b
 In other words, we have gone back to the primitive age of middle school students cursing each other, saying stuff like 'boys are terrible' and 'girls are annoying'.
 <Page22>
 She looks at the situation with a warm smile and says,
-"Ufufu. I wonder if these kids are alright in their heads."
+"Ufufu. I wonder if these kids are alright in the head." //Reads better following the expression "right in the head."
 The new teacher makes a scathing remark.
 <Page23>
 I couldn't keep up with the atmosphere of the classroom so I looked away, making eye contact with Yumizuka in the process. She seems not to be in it as well, looking as absent-minded as I am.
 ...What is this atmosphere?
-The classroom is abuzz with color. What seemed to be a peaceful day has taken an unexpected turn.
+The classroom is full of color. What seemed to be a peaceful day has taken an unexpected turn. //"Buzzing with color" mixes sound and sight. I used "full of color" here, but "glowing with color" would also work.
```

Comment:
I feel like I saw this line get fixed in a @Gekimon PR recently, did I imagine it or are there 2 files with similar text?